app/controllers/ember-community-survey-2020.js

```diff
@@ -269,35 +243,6 @@ const SS_Q302 = {
   ],
 };
 
-// Q3.03: At this time next year, how many Ember developers does your company plan to employ?
-//------------------------------------------------
-
-const SS_Q303 = {
-  options: {
-    chart: { type: 'column' },
-    title: {
-      text:
-        'At this time next year, how many Ember developers does your company plan to employ?',
-    },
-    subtitle: { text: '' },
-    tooltip: { pointFormat: '{point.y:.2f}%' },
-    xAxis: { type: 'category' },
-    yAxis: { title: { text: 'Percent' } },
-  },
-  data: [
-    {
-      name: '2019',
-      color: color2019,
-      data: [
-        { name: 'More', y: (100 / 1006) * 530 },
```

Comment:
For some reason, the denominator `1006` in the year 2020 didn't match the denominator `1232` in the year 2019, even though the numerators (the actual counts) did match.

Not sure why we "lost" some number of survey responses. This happened in several other places in the 2020 file.